package.json

```diff
@@ -100,6 +104,8 @@
     "es6-promise": "^3.0.2",
     "file-saver": "^1.3.3",
     "font-awesome": "^4.5.0",
+    "glsl-parser": "git+https://github.com/anvaka/glslx.git#glsl-parser",
+    "glslify": "^6.1.1",
```

Comment:
I didn't see this being used in the project, is it needed explicitly for glsl-parser?

Reply:
Good catch, this is obsolete.